apps/site/pages/docs/customizing/components.mdx

```diff
@@ -1,13 +1,23 @@
-import { Callout } from 'nextra-theme-docs'
+import { Callout, Steps } from 'nextra-theme-docs'
```

Comment:
I think we can rename the whole section to `Styling Components` instead of Components. WTDY?
<img width="371" alt="image" src="https://github.com/vtex/faststore/assets/3356699/b29fc8e3-9046-48f3-b052-f648138253d8">